nbdime/utils.py

```diff
@@ -49,15 +51,18 @@ def join_path(*args):
     return ret if ret.startswith("/") else "/" + ret
 
 
+r_is_int = re.compile(r"^[-+]?\d+$")
+
 def star_path(path):
-    """Replace integers and numeric strings in a path with * """
+    """Replace integers and decimal strings in a path with * """
```

Comment:
Maybe say "integer strings"? Decimal implies float numbers will be replaced as well.